src/form-generator/form-handler-one-step.ts

```diff
@@ -64,6 +82,64 @@ export class FormHandlerOneStep {
     return containerButton;
   };
 
+  createSendButtonContainer = (container: any, apiKeyWeb3Forms: string): any => {
```

Comment:
znowu ta metoda za duzo robi lub nie robi to co mowi ze roib.
createSendButtonContainer. -> to znaczy stworz kontener na button do wysylki formularza
A tu ja widze wyrzucanie jakis danych z formularza.
Ta metoda robi tak : 
- wywoluje metode tworzaca button - okej 
- znowu jest showSuccessMessage - ktore tworzy popup z message sukcesu. Show i Create to dwie rozne rzeczy. bo stworzyc popup a go pokazac to dwie rozne rzeczy 
- usuwasz jakies dane z formularza
- calluijesz api zeby wziac response 
- itd - nie chce mi sie pisac wiecej
Ale wielokrotnie mowilem o tym ze metoda musi miec jedna odpowiedzialnosc. Ten kod robi tak wiele rzeczy i nazwa jest tak mylaca ze to nei moze tak zostac. 
Trzeba z tego wydzielic kolejne pewnie 10 metod
ZMienic nazwe glownej , nazwy zmiennych itd
Bo chcac nie chcac ta metoda to dramat